src/common/chat-participants/powerpages/PowerPagesChatParticipantConstants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+export const POWERPAGES_CHAT_PARTICIPANT_ID = 'powerpages';
+
```

Comment:
nit: dont leave extra space - unless you are sectioning them  - in which case leave a comment on top explaining what those section of const are about